src/sandbox.c

```diff
@@ -252,7 +254,7 @@ int main(int argc, char **argv)
 		dputs("Setting up the required environment variables.");
 
 	/* If not in portage, cd into it work directory */
-	if ('\0' != sandbox_info.work_dir[0])
+	if (strlen(sandbox_info.work_dir))
```

Comment:
What is the reason for this change? The existing check is slightly faster.

Reply:
Nowadays strlen can be assumed to be implemented by a compiler builtin, and principally I'm  inclined to prefer readability over some minimal speed gain at the price of a barbed-wire look&feel, at least in code sections that run only once during initialization. But this assumption seems not be fully justified with sandbox, as the authors provide their own implementation of memory and string routines like strlen in C, for a rather sensible reason.

Reply:
I don't think sandbox re-implements `strlen`; where did you see that?

Readability is certain a valid reason to change this.